### PR TITLE
Circleci project setup

### PR DESCRIPTION
**Pull request by coryschwartz**
Setup deployment into the current fil-infra clusters.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM golang:alpine as builder
+FROM golang:1.17-alpine as builder
```
Comment (coryschwartz) [inline Dockerfile:1]: Something changed since  I built this. golang:alpine complains that git isn't installed.

**Comment by coryschwartz** [comment]
@mcamou currently I just deployed to us-east-2-dev

we can of course add the others to this as well.

https://app.circleci.com/pipelines/github/filecoin-project/lotus-infra/1629/workflows/d9f44153-586e-45f9-9870-b955502fe0db/jobs/1771

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+# See: https://circleci.com/docs/2.0/configuration-reference/#workflows
+workflows:
+  deploy: # This is the name of the workflow, feel free to change it to better match your workflow.
+    # Inside the workflow, you define the jobs you want to run.
+    jobs:
+      - build-push:
+          context:
+            - ipfsshipyardbot-docker
+      - deploy-lotusinfra:
+          name: deploy-us-east-2-dev
```
Comment (mcamou) [inline .circleci/config.yml:72]: We're only deploying to `us-east-2`? I thought that we had it running in all geos where we have cluster hosts

File: Dockerfile (hunk 1 above)
Comment (mcamou) [reply Dockerfile:1]: I added `git` to `apk add` on my PR, but this is better. I am a firm believer in locking down versions.

File: .circleci/config.yml (hunk 2 above)
Comment (coryschwartz) [reply .circleci/config.yml:72]: I fixed this -- added the other clusters.

**Review by mcamou — APPROVED** [review]

**Pull request merged**